src/Service.Host/client/src/itemTypes.js

```diff
@@ -46,10 +46,10 @@ export const salesArticles = new ItemType(
     'SALES_ARTICLES',
     '/products/maint/sales-articles'
 );
-export const salesArticlesReallocate = new ItemType(
-    'salesArticlesReallocate',
-    'SALES_ARTICLES_REALLOCATE',
-    '/products/maint/sales-articles-reallocate'
+export const TariffsReallocate = new ItemType(
```

Comment:
this should have a lowercase t as in tariffsReallocate. 

itemType names need to perfectly match the reducer names for a few things to work correctly.

If you fix that (and update any references of course) then that should solve your error messages problem.

Reply:
Thank you!!!! Think I was just too late in the day to spot the capital there, that of course did allow me to get the errors displaying properly :)